tests/Feature/TasksTest.php

```diff
@@ -37,23 +19,13 @@ public function testIndex() {
         $response->assertSee('class="container" id="tasks"'); // does the string show within the view?
     }
 
-    // public function testEdit()
-    // {
-    //   $tasks_uuid = factory(User::class)->create()->uuid_text;
-    //   $response = $this->webGet('tasks/'.$tasks_uuid . '/edit');
-    // //   $response->assertStatus(200);
-    //   $response->assertViewIs('tasks.edit');
-    // }
-
-    // public function testCreateRoute()
-    // {
 
-    //   // get the URL
-    //   $response = $this->apiCall('GET', '/admin/users/create');
-
-    //   $response->assertStatus(200);
-    //   // check the correct view is called
-    //   $response->assertViewIs('admin.users.create');
-
-    // }
+    public function testShow() 
+    {
+        // $task = factory(Task::class)->create(['name'=>'Test show']); 
+        $response = $this->webGet('tasks/id123');
```

Comment:
Where is this ID coming from?

Reply:
Nolan and I created a fake ID as there is no Task Model to generate faker info